### PR TITLE
feat: add autofix for rule "selector-type-case"

### LINKED ISSUE #3663: Add autofix to selector-type-case
> What is the problem you're trying to solve?

Adds autofix of rules [defined in `stylelint-config-standard`](https://github.com/stylelint/stylelint-config-standard/blob/master/index.js) so that everyone can start using [`stylelint-config-standard`](https://github.com/stylelint/stylelint-config-standard) easily.

> What solution would you like to see?

Add autofix to [selector-type-case](https://github.com/stylelint/stylelint/tree/master/lib/rules/selector-type-case)

### PR DESCRIPTION
**Pull request by EliazTray**
> Which issue, if any, is this issue related to?

e.g. "Closes #3663 "

> Is there anything in the PR that needs further explanation?

e.g. "No, it's self explanatory."

### CODE REVIEW

**Comment by EliazTray** [comment]
@ota-meshi 

in branch master,  it asls occurs [this](https://travis-ci.org/stylelint/stylelint/jobs/428159536) when run test.  how to solve it?

**Comment by ota-meshi** [comment]
I think it is the same problem as [here](https://github.com/stylelint/stylelint/issues/3638#issuecomment-419738905).

**Comment by ota-meshi** [comment]
@EliazTray 
> in branch master, it asls occurs this when run test. how to solve it?

It was solved!

File: lib/rules/selector-type-case/__tests__/index.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const { messages, ruleName } = rule;
 testRule(rule, {
   ruleName,
   config: ["lower"],
+  fix: true,
```
[the remark this answers is not in the document]
Comment (EliazTray) [reply lib/rules/selector-type-case/__tests__/index.js]: Could you give an example ?

[the remark this answers is not in the document]
Comment (ota-meshi) [reply lib/rules/selector-type-case/__tests__/index.js]: For example css below.

```css
a, /* comment */
b {}
```

[the remark this answers is not in the document]
Comment (EliazTray) [reply lib/rules/selector-type-case/__tests__/index.js]: i see, 👌

**Comment by ota-meshi** [comment]
@EliazTray Thank you for excellent work!
but, I should have requested one more thing, but I overlooked it. sorry...

Could you add the following sentence to [README.md](https://github.com/stylelint/stylelint/blob/master/lib/rules/selector-type-case/README.md)?

```md
The `--fix` option on the [command line](../../../docs/user-guide/cli.md#autofixing-errors) can automatically fix all of the problems reported by this rule.
```

Please refer to [here](https://github.com/stylelint/stylelint/blob/master/lib/rules/color-hex-case/README.md).

**Comment by EliazTray** [comment]
@ota-meshi 
Thanks for your reviewing!
I had added the sentence and rebased the history commits.

**Comment by jeddy3** [comment]
> -   Added: `selector-type-case` autofix ([#3668](https://github.com/stylelint/stylelint/pull/3668)).

**Pull request merged**